site/source/docs/tools_reference/emsdk.rst

```diff
@@ -104,7 +104,7 @@ the variable names used to point to the different tools::
   # .emscripten file from Linux SDK
 
   import os
-  NODE_JS = 'nodejs'
+  NODE_JS='nodejs'
   LLVM_ROOT='/home/ubuntu/emsdk/upstream/bin'
```

Comment:
actually since this is python code lest use spaces about both assignment operators maybe?

Reply:
oh, i mistook this file is a shell, so that i remove the space around assignment operator, i'll revert it.